perun-core/src/main/java/cz/metacentrum/perun/core/entry/AttributesManagerEntry.java

```diff
@@ -1805,6 +1906,8 @@ public void deleteAttribute(PerunSession sess, AttributeDefinition attributeDefi
 		getPerunBl().getMembersManagerBl().checkMemberExists(sess, member);
 		List<Attribute> attributes = getAttributesManagerBl().getRequiredAttributes(sess, service, resource, member, workWithUserAttributes);
 		Iterator<Attribute> attrIter = attributes.iterator();
+		User u = null;
```

Comment:
Please use full name of variable "user" instead of just "u" when you are already working with this object. And also please call it from this place always. It is just one calling of method getUserByMember and it will be more human readable.